sunpy/net/jsoc/jsoc.py

```diff
@@ -323,7 +324,7 @@ def search(self, *query, **kwargs):
         """
 
         return_results = JSOCResponse()
-        query = and_(*query)
+        query = and_(*(*query, attrs.Quality('?QUALITY >= 0?')))
```

Comment:
I don't really know enough about this, but should we only be asking the user to specify the bit after `QUALITY` i.e. you would do something like `a.jsoc.Quality(">= 0")`?

Reply:
Yea me neither AFAIK they are defined as bitflags and then you'd need convert to number and pass that in. I'm assuming there is a convention that if the data is missing the corresponding number is negative?

And the exact meaning seem to be dependant on the series and level i.e. http://jsoc.stanford.edu/jsocwiki/Lev1Doc

Reply:
oh goodness, I can totally read that C code. :eyes: 

The `?QUALITY` bit of the query is always going to be the same right? If we leave the specifier section freeform we shouldn't be restricting what people can ask for?